src/utils/wallet.js

```diff
@@ -41,17 +41,15 @@ export const groomingTransactionData = (transaction, steemPerMVests, formatNumbe
 
       const { author, permlink } = opData;
 
-      sbdPayout = formatNumber(parseToken(sbdPayout), { minimumFractionDigits: 3 }).replace(
-        ',',
-        '.',
-      );
-      steemPayout = formatNumber(parseToken(steemPayout), { minimumFractionDigits: 3 }).replace(
-        ',',
-        '.',
-      );
-      vestingPayout = formatNumber(vestsToSp(parseToken(vestingPayout), steemPerMVests), {
-        minimumFractionDigits: 3,
-      }).replace(',', '.');
```

Comment:
i guess we was use `formatNumber ` for a issue, i couldnt remember what was the issue. Why did you remove to that

Reply:
On Android Nexus S API 24, numbers were not correctly formatted, long numbers in curation and author rewards. These was reported by users multiple times and I couldn't find root cause other than formatNumber not working properly on older Android devices, so it is simple Floating point change, so `.toFixed()` should do the job just fine.

Reply:
hmm okay we can try this solution